backend/setup.py

```diff
@@ -39,7 +39,7 @@
     # mail-notifier
     "mako",
     "lxml>=4.8.0",
-    "redis>=3.5.3",
```

Comment:
Question: do we have same version like at present? or 4.6.0 is not the latest but necessary version.

Reply:
4.6.0 is the latest update of the highest major version that supports python 3.7, This will be updated once we drop python 3.7 support